src/LocalImage/AbstractLocalImage.php

```diff
@@ -21,14 +21,14 @@ abstract class AbstractLocalImage
      *
      * @var mixed|null
      */
-    protected $context = null;
+    protected $context;
 
     /**
      * The rokka hash from the rokka API.
      *
      * @var string|null
      */
-    protected $rokkaHash = null;
+    protected $rokkaHash;
```

Comment:
this is not initialized in the constructor, we should add its initialization too

Reply:
that is the cs fixer. removing redundant code when there is no type, it is automatically initialized to `null`. when we add property types, we will need to re-add the ` = null` but we should only do that in a major version.